games/ShootEmUp/scripts/Camera2D.gd

```diff
@@ -0,0 +1,20 @@
+extends Camera2D
+
+var shake_strength: float = 0.0
+
+const SHAKE_DURATION: float = 5.0
+const DEFAULT_SCREEN_SHAKE: float = 30.0
+
+func shake_screen(strength: int = 0) -> void:
+	if strength == 0:
+		shake_strength = DEFAULT_SCREEN_SHAKE
+	else:
+		shake_strength = strength
+	
+func _process(delta):	
+	if shake_strength > 0:
+		shake_strength = lerpf(shake_strength, 0 , SHAKE_DURATION * delta)
```

Comment:
It should be noted that if either `delta` or `SHAKE_DURATION` are too high (such that `SHAKE_DURATION * delta > 1`), this could end up overshooting the 0 mark and end up in the negatives. Thus, perhaps we should min this value at 0, or max `SHAKE_DURATION * delta` at 1, like so:

```suggestion
		shake_strength = lerpf(shake_strength, 0 , shake_decay * delta)
		shake_strength = max(shake_strength, 0)
```
(I also added the above suggested name change in there)